bundles/org.platformio.eclipse.ide.home.ui/src/org/platformio/eclipse/ide/home/internal/ui/PerspectiveFactory.java

```diff
@@ -27,10 +27,13 @@
 
 public final class PerspectiveFactory implements IPerspectiveFactory {
 
+	private static final String ID_CONSOLE = "org.eclipse.ui.console.ConsoleView"; //$NON-NLS-1$
```

Comment:
This is old good Eclipse style
But currently we would prefer 
`private final String console = "org.eclipse.ui.console.ConsoleView"; //$NON-NLS-1$`